multiapps-controller-api/pom.xml

```diff
@@ -128,6 +130,11 @@
                                 <apiSource>
                                     <locations>org.cloudfoundry.multiapps.controller.api.v2</locations>
                                     <swaggerFileName>mtarest_v2</swaggerFileName>
+                                    <info>                                                
```

Comment:
Why there is no basePath field for api v2?

Reply:
removed the other one as well, as swagger does not consider that  parameter correctly - the paths still contain the base path